.github/workflows/publish-build.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 Practically.io All rights reserved
+
+name: Publish Build
+
+on:
+  push: {branches: ['1.x']}
+  workflow_run:
+    workflows:
+      - 'CI'
+    types:
+      - 'completed'
```

Comment:
This still runs when tie CI action files you will need to add a guard for failed pipelines, This is an issue I have been having with my publishing to NPM job. I think we can:

1) Add a run-if to the job to only run when the CI is successful
2) Use a trigger to trigger this when CI is successful.

Reply:
Yup happy with that :+1: